test/javascript/components/magellan.js

```diff
@@ -56,7 +56,9 @@ describe('Magellan', function() {
 
       // Jump to last section
       var target = $html.find('a').eq(-1).attr('href');
-      plugin.scrollToLoc(target);      
+      $html.one('load', function(){
+        plugin.scrollToLoc(target);
+			});
```

Comment:
The function here is executed dynamically, far after the test function exited. So the test happends before the scroll begin and therefore does not test it.

Reply:
A solution would be to wait for `$html.one('load'` to execute the test (and everything with it). I wonder if we can use `async await` in tests (for a simpler code)...

Reply:
Ah you mean the `done`callback.

Reply:
I forgot the `done` callback but yes.

```js
// .... initialize plugin

// wait for page to load
$html.one('load', function(){
  // scroll
  plugin.scrollToLoc(target);
  
  // wait for the scroll to takes effect
  setTimeout(function() {
    // ... test everything ...
    // and say we are done
    done();
  });
});
```

But as you see, this is a bit nested. It would be simpler with async/await

Reply:
`async await` is not supported in all browsers, for example IE11

Reply:
`async await` is supported almost nowhere. My point is that test files are transpiled (see [gulp/tasks/test.js:35](https://github.com/zurb/foundation-sites/blob/develop/gulp/tasks/test.js#L35)). But our current global babel config ([.babelrc](https://github.com/zurb/foundation-sites/blob/DanielRuf/fix/develop-dropdown-eventnames/.babelrc)) does not allow `async/await` (see https://medium.com/@dandv/using-es2016-es7-async-await-with-babel-e758bab1c4d5).